crypto/bio/bss_file.c

```diff
@@ -198,7 +198,7 @@ BIO *BIO_new_fp(FILE *stream, int close_flag)
         return (NULL);
 
     BIO_set_flags(ret, BIO_FLAGS_UPLINK); /* redundant, left for
```

Comment:
If you're changing tihs, make it two lines with the comment a single line before the ```BIO_set_flags``` call.